README.md

```diff
@@ -1 +1,210 @@
+
+### Prerequisites
+
+In order to run this project you need:
+
+```sh
+ To have a computer, Internet, Keyboard and Mouse
+```
+
+### Setup
+
+Clone this repository to your desired folder:
+
+```sh
+  Open it with Visual Studio Code, and open a server with "LiveServer".
+```
+
+### Install
+
+Install this project with:
+
+```sh
+  Installation is not necessary
+```
+
+### Usage
+
+To run the project, execute the following command:
+
+```sh
+Open Live Server
+```
+
+### Run tests
+
+To run tests, run the following command:
+
+```sh
+  To check the HTML functionality use: 'HTML npx hint .'
+```
+
+```sh
+  To check the CSS functionality use: 'npx stylelint "**/*.{css,scss}"'
+```
+
+### Deployment
+
+You can deploy this project using:
+
+```sh
+Your software of preference
+```
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHOR -->
+
+## 👥 Author <a name="author"></a>
+
+👤 **Author**
+
+- GitHub: [@VelzckC0D3](https://github.com/VelzckC0D3)
+- LinkedIn: [VelzckC0D3](https://www.linkedin.com/in/velzckcode/)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FUTURE FEATURES -->
+
+## 🔭 Future Features <a name="future-features"></a>
+
+- [ ] **[CSS Animations and Transitions]**
+- [ ] **[Easy user interface to navigate with]**
+- [ ] **[Complete, user friendly design, responsive and optimized]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- CONTRIBUTING -->
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- SUPPORT -->
+
+## ⭐️ Show your support <a name="support"></a>
+
+If you like this project, be pending on my profile since I'll be doing much more! 
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- ACKNOWLEDGEMENTS -->
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+I would like to thanks my Microverse Team for helping me to get this done.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FAQ (optional) -->
+
+## ❓ FAQ<a name="faq"></a>
+
+- **[Which languages did you use in this project?]**
+
+  - [only HTML and CSS]
+
+- **[How were you able to do the Icons hover effect?]**
+
+  - [It's a round solid border, with a transparent color that changes on a smooth transition to orange when you hover it.]
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LICENSE -->
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./LICENSE) licensed.
+
+_NOTE: we recommend using the [MIT license](https://choosealicense.com/licenses/mit/) - you can set it up quickly by [using templates available on GitHub](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-license-to-a-repository). You can also use [any other license](https://choosealicense.com/licenses/) if you wish._
```

Comment:
[Optional] Kindly remove this from your readme file. It is only an instruction to help you know what to do in this section and should not be part of the final readme😎